src/remixAppManager.js

```diff
@@ -179,6 +179,17 @@ export class RemixAppManager extends PluginEngine {
       icon: 'https://image.flaticon.com/icons/svg/1570/1570493.svg',
       location: 'sidePanel'
     }
+    const debugPlugin = {
+      name: 'debugPlugin',
+      displayName: 'Debug Tools for Remix plugins',
+      description: 'Easily test and debug your plugins !',
+      methods: ['sayHello', 'sayMyName', 'sayOurNames'], // test calls with 0, 1, and 2 args
+      events: [],
+      version: '0.1.0-alpha',
+      url: 'https://remix-debug-a.surge.sh',
+      icon: 'https://remix-debug-a.surge.sh/icon.png',
```

Comment:
we use a very similar icon for our native Debug plugin. both will be shown on Vertical panel when activated.

Reply:
Yeah I wasn't sure about my icon, I try to mix the debug icon with the plugin manager icon, I can change that without changing the icon's url. Also I'm open to any icon suggestion

Reply:
https://fontawesome.com/icons/spider?style=solid ? :) I'm not sure either, but it should be different

Reply:
I'll do it this evening

Reply:
Hi ! I have changed the icon, what do you think now ?
![debug plugin icon](https://remix-debug-a.surge.sh/icon.png)

Reply:
I like it :))